app/views/records/edit_fields/_qualification_name.html.erb

```diff
@@ -1,8 +1,7 @@
 <% qualification_name_service = HykuAddons::QualificationNameService.new %>
 <% selected =  f.object[:qualification_name].empty? ? nil :  f.object[:qualification_name] %>
-<% set_default = selected || 'PhD' %>
 
 <% model = f.object.model.class.to_s.underscore %>
 
 <%= render 'hyku_addons/qualification/qualification', f: f, model: model, qualification_service: qualification_name_service,
-  set_default: set_default, field_name: 'qualification_name', css_class_name: 'qualification-name'%>
+    field_name: 'qualification_name', css_class_name: 'qualification-name'%>
```

Comment:
Includes like this shouldn't be broken onto a newline at a random index. Instead, try moving the argument into a seperate hash: 

```
<% options = { 
    f: f, 
    model: model, 
    qualification_service: qualification_name_service, 
    field_name: 'qualification_name', 
    css_class_name: 'qualification-name' 
} %>
<%=render 'hyku_addons/qualification/qualification', options %>